LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) Praekelt Foundation and individual contributors.
```

Comment:
I'm used to seeing a date here, like this: `Copyright (c) 2016, Praekelt Foundation`, not sure how important that is.